src/commons/axiosInstance.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
```

Comment:
Thank you for separating this concern so that the team can just call it later.

Reply:
I checked it out on Heroku review apps and we'll need to test out facebook probably after the App is no longer in development mode.